cassiopeia/solver/solver_utilities.py

```diff
@@ -122,3 +123,24 @@ def convert_sample_names_to_indices(
     name_to_index = dict(zip(names, range(len(names))))
 
     return list(map(lambda x: name_to_index[x], samples))
+
+def save_dissimilarity_as_phylip(
```

Comment:
we should add a test for this, it can be under `test/solver_tests/ccphylo_solver_test.py` if that's easiest.

Reply:
Still need a test for this function.